crawl-ref/source/xom.cc

```diff
@@ -2007,7 +2007,18 @@ static void _xom_pseudo_miscast(int /*sever*/)
     }
 
     if (you.slot_item(EQ_CLOAK))
-        messages.emplace_back("Your cloak billows in an unfelt wind.");
+    {
+        item_def* item = you.slot_item(EQ_CLOAK);
+
+        if(item->sub_type == ARM_CLOAK)
+        {
+            messages.emplace_back("Your cloak billows in an unfelt wind.");
+        }
+        else if(item->sub_type == ARM_SCARF)
+        {
+            messages.emplace_back("Your scarf briefly wraps itself around your head!");
```

Comment:
Thanks for the patch, good catch on this messaging issue.

Ridiculously picky comments: the house bracketing style would have no curly braces in these because the bodies of the if clauses are one line (you can use the util/unbrace command to enforce this) and we normally have a space after "if".

But this change looks good to me and if you don't have a chance to do these tweaks I (or someone else if they want) can easily add them and merge this.

Reply:
Fixed!